cloud/storage/core/tools/ci/runner/run_all.sh

```diff
@@ -17,9 +17,11 @@ fi
 cd $nbspath &&
 git pull
 
-logs_dir="/var/www/build/logs/run_$(date +%y_%m_%d__%H)" &&
+logs_root="/var/www/build/logs"
+logs_dir="${logs_root}/run_$(date +%y_%m_%d__%H)" &&
 rm -rf "$logs_dir" &&
 mkdir -p "$logs_dir"
+find  "${logs_root}" -maxdepth 1 -mtime +30 -type d -exec rm -rf {} \;
```

Comment:
https://unix.stackexchange.com/a/167824

Reply:
-delete does not work on directories. -delete for directories implies depth.